Frontend/CO.CDP.OrganisationApp/CO.CDP.OrganisationApp.csproj

```diff
@@ -34,12 +34,6 @@
         <ProjectReference Include="..\..\Services\CO.CDP.Localization\CO.CDP.Localization.csproj" />
     </ItemGroup>
 
-    <ItemGroup>
-        <None Update="mou-pdfs\version-1.pdf">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-    </ItemGroup>
-
```

Comment:
I see you've moved the MoU into www-root... Which would be fine in principle, but I think this will error in the `OnGetDownload` method of `Frontend\CO.CDP.OrganisationApp\Pages\MoU\ReviewAndSignMemorandom.cshtml.cs`

This loads the file up from disk and streams it out.

Also, if you look at this method that streams it out, it gives it a name dynamically on the fly, so whatever version of the MoU it currently is, it'll still be called `fts-joint-controller-agreement.pdf`

I think there's a couple of options:

1) Rework the email so that it points to this `OnGetDownload` action and just reuse that.

2) Consider - should the email download the version of the MoU that they _signed_? Probably yes? So we'd need a version of `OnGetDownload` that let you grab the specific version?